### PR TITLE
chore: zksync-ethers 5.8.0-beta.5

### PR DESCRIPTION
**Pull request by benceharomi**
## What ❔

Start using the https://github.com/zksync-sdk/zksync-ethers/releases/tag/v5.8.0-beta.5 **release** from the SDK, instead of depending a GitHub commit.

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.
- [ ] Linkcheck has been run via `zk linkcheck`.

### CODE REVIEW

**Comment by mm-zk** [comment]
LGTM - but please revert the change in contracts (the code should work without it, right?)

File: yarn.lock [hunk 1]
```diff
@@ -11010,19 +11010,20 @@ yocto-queue@^1.0.0:
...
-  resolved "https://github.com/zksync-sdk/zksync-ethers#28ccbe7d67b170c202b17475e06a82002e6e3acc"
+zksync-ethers@5.8.0-beta.5:
+  version "5.8.0-beta.5"
+  resolved "https://registry.yarnpkg.com/zksync-ethers/-/zksync-ethers-5.8.0-beta.5.tgz#4f70193a86bd1e41b25b0aa5aa32f6d41d52f7c6"
+  integrity sha512-saT/3OwLgifqzrBG7OujvUMapzXnshAaLzAZMycUtdV20eLSSVkyLIARVwh1M6hMQIUvX2htV0JN82QRMyM3Ig==
   dependencies:
     ethers "~5.7.0"
 
-"zksync-ethers@https://github.com/zksync-sdk/zksync-ethers#sb-old-sdk":
+"zksync-ethers@https://github.com/zksync-sdk/zksync-ethers#ethers-v5-feat/bridgehub":
```
Comment (mm-zk) [inline yarn.lock:11020]: do we still use this one for something?

File: yarn.lock [hunk 2]
```diff
@@ -11010,19 +11010,20 @@ yocto-queue@^1.0.0:
...
   version "5.1.0"
-  resolved "https://github.com/zksync-sdk/zksync-ethers#7e5e8edc57c848aabdb4b018de8270addc82aa6e"
+  resolved "https://github.com/zksync-sdk/zksync-ethers#28ccbe7d67b170c202b17475e06a82002e6e3acc"
   dependencies:
     ethers "~5.7.0"
 
-zksync-web3@^0.15.4, zksync-web3@^0.15.5:
+zksync-web3@^0.15.4:
```
Comment (mm-zk) [inline yarn.lock:11026]: and this one is still used somewhere?

**Review by mm-zk — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (benceharomi) [reply yarn.lock:11020]: It is used by contracts, in zksync-era everything is upgraded to 5.8.0-beta.5

File: yarn.lock (hunk 2 above)
Comment (benceharomi) [reply yarn.lock:11026]: and the same applies here

**Pull request merged**